specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/CommonDefinitions.json

```diff
@@ -429,6 +433,10 @@
           "x-ms-identifiers": [
             "triggerParameter"
           ]
+        },
+        "identity": {
```

Comment:
Not blocking your change - Could this be moved to base definition and referenced for all scale rules